transaction-context/src/lib.rs

```diff
@@ -49,21 +49,22 @@ static_assertions::const_assert_eq!(
     solana_account_info::MAX_PERMITTED_DATA_INCREASE
 );
 
+pub const MAX_ACCOUNTS_PER_TRANSACTION: usize = 256;
```

Comment:
Best to static assert this value with dev-dependencies / debug mode.

Reply:
There is no constant for the maximum number of accounts in a transaction. The closest one is this one:

https://github.com/anza-xyz/agave/blob/0725eb32b2e21ff6309d11613706bb8b18a77589/syscalls/src/cpi.rs#L17

which only affects CPI. That does not prevent a `CompiledInstruciton` or a `SVMInstruction` to refer to an account with index 255.